diode-proto/diode/v1/reconciler.proto

```diff
@@ -61,6 +61,15 @@ enum State {
   NO_CHANGES = 4;
 }
 
+// Log Stats
+message Stats {
```

Comment:
Shall we be more explicit with naming, e.g. `IngestionStats` or even `IngestionMetrics`, thoughts?

Reply:
yes, I'm fine changing it to either

Reply:
Changed to Ingestion Metrics